makes/utils/makePythonLock/update.sh

```diff
@@ -3,7 +3,7 @@
 function main {
   local cmd=(just m . /utils/makePythonLock)
 
-  : \
+  : \  \
```

Comment:
Typo?

Reply:
No, It was added by `formatBash`

Reply:
Maybe making  the first command look like `: && other \` would make this look better 🤔

Reply:
On it!